diffkemp/diffkemp.py

```diff
@@ -654,6 +670,9 @@ def view(args):
         yaml.dump(yaml_result, file, sort_keys=False)
 
     if args.devel:
-        os.system(f"cd {VIEW_DIRECTORY} && npm start")
+        os.system(f"cd {view_directory} && npm start")
+    # run serve directly if diffkemp is installed
+    elif view_directory == VIEW_INSTALL_DIR:
+        os.system(f"npx serve {PUBLIC_DIRECTORY}")
     else:
-        os.system(f"cd {VIEW_DIRECTORY} && npm run prod")
+        os.system(f"cd {view_directory} && npm start")
```

Comment:
Why not `npm run prod` as was here before?

Reply:
Not relevant after #280 is merged.

Reply:
Thanks, you were right, it was my mistake...